client/src/Core/Routes.js

```diff
@@ -1,31 +1,63 @@
 //routing files
-import React, {lazy, Suspense} from 'react';
-// import '../Assets/Styles/Common/style.css';
-import {RENDER_URL} from '../Utils/Urls';
-import { BrowserRouter as Router, Switch, Route} from 'react-router-dom';
-const AuthenticatedLandingComponent =  lazy(() => import('../Modules/AuthenticatedLanding/AuthenticatedLandingComponent'));
-const ConfirmationComponent = lazy(() => import("../Modules/Confirmation/ConfirmationComponent"));
-const DashBoardContainer = lazy(() => import("../Modules/Dashboard/DashBoardContainer"));
-/*const EventContainer = lazy(() => import("../Modules/Events/EventContainer"));
-const EventListContainer = lazy(() => import("../Modules/Events/EventListContaner"));*/
-const EventListComponent = lazy(() => import("../Modules/Events/EventContainer"));
+import React, { lazy, Suspense } from "react";
+import { RENDER_URL } from "../Utils/Urls";
+import "../Assets/scss/main.scss";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
+const ConfirmationComponent = lazy(() =>
+  import("../Modules/Confirmation/ConfirmationComponent")
+);
+const DashBoardContainer = lazy(() =>
+  import("../Modules/Dashboard/DashBoardContainer")
+);
+const EventContainer = lazy(() => import("../Modules/Events/EventContainer"));
 const FamilyContainer = lazy(() => import("../Modules/Family/FamilyContainer"));
-const App = lazy(() => import("../App"));
+const WrapperComponent = lazy(() =>
+  import("../Modules/General/WrapperComponent")
+);
 
 const Routes = () => {
-    return (
-        <Router basename="/">
```

Comment:
The Router basename is "/" as default. If we need to specify all routes to take a prefix, we need to mention basename. For example, basename="/app". So that every other route will be called as "/app/events/list", "/app/events/1/confirm" etc.

Reply:
Good catch. That basename is required as all out routing is done by `exact`. React encounters our `whatever.beta.index.html `is confused in our environment. It'll need to be put back in.

Reply:
Ok. I will add it back on the next PR.